source/global-this.d.ts

```diff
@@ -0,0 +1,21 @@
+/**
+Type for `globalThis`. Extend to declare locally scoped global symbols.
+
+When defining a global symbol in a declaration file is inappropriate, it can be helpful to define an `interface` or `type` (say `Foo`) and then cast `globalThis` via code like `globalThis as unknown as Foo`.
```

Comment:
I would use a different word than "symbol" as symbol has a specific meaning in JS that does not apply to this.

Reply:
Updated.